man/rmd-fragments/caveats.Rmd

```diff
@@ -0,0 +1,13 @@
+### Edge cases
+
+* If you **rebase after creating a tag**, you need to call `finalize_version()` to make sure the tag is moved to the mainline branch.
+
+* If you **pushed after calling `finalize_version()` (directly or indirectly)**, the tag may have been pushed as well. In this case, invoke
```

Comment:
maybe add examples for directly and indirectly?

Reply:
```suggestion
* If you **pushed** via `finalize_version(push = TRUE)` or `git push`, the tag may have been pushed as well. In this case, invoke
```